doc/radosgw/s3/commons.rst

```diff
@@ -18,7 +18,7 @@ The second method identifies the bucket via a virtual bucket host name. For exam
 	GET / HTTP/1.1
 	Host: mybucket.cname.domain.com
 
-To configure virtual hosted buckets, you can either set ``rgw_dns_name = cname.domain.com`` in ceph.conf, or add ``cname.domain.com`` to the list of ``hostnames`` in your zonegroup configuration. See `Ceph Object Gateway - Multisite Configuration`_ for more on zonegroups.
+To configure virtual hosted buckets, you can either set ``rgw_dns_name = cname.domain.com,cname2.domain2.com`` in ceph.conf, or add ``cname.domain.com`` and ``cname2.domain2.com`` to the list of ``hostnames`` in your zonegroup configuration. See `Ceph Object Gateway - Multisite Configuration`_ for more on zonegroups.
```

Comment:
this is meant to be a simple example for enabling virtual-hosted access to buckets, so i don't think it's helpful to introduce an extra `cname2.domain2.com` here

instead, maybe just add a `tip` or `note` below that mentions support for multiple dns names and links to the config option https://docs.ceph.com/en/latest/radosgw/config-ref/#confval-rgw_dns_name like:
```
:confval:`rgw_dns_name`
```

Reply:
jenkins render docs